### DISCUSSION TITLE
Using with TA-lib

### DISCUSSION THREAD

**Discussion by Arx-Wind** [kernc/backtesting.py#372] [Q&A]
This may seem like a dumb question but it would be great if someone could help me out …How do I use this module with TA-lib indicator ? I want to backtest a strategy with ADX indicator but for the life of me can’t figure out how to import ADX indicator with TA-lib . The guide documentation  only shows how to use it with the inbuilt SMA . 
thanks in advance to anyone kind enough to respond

**Comment by kernc** [comment]
Have you tried passing `ta.ADX` as the `func` to [`Strategy.I()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I)?
```py
import talib as ta

...

        self.I(ta.ADX, ...)
```

**Comment by Arx-Wind** [comment]
Wow! That helped . Thank you so much for the response . Much appreciated. Thank you .

**Comment by cce15** [comment]
Thank you so much
